backend/api/Program.cs

```diff
@@ -73,7 +73,7 @@
 builder.Services.AddScoped<ITaskDurationService, TaskDurationService>();
 builder.Services.AddScoped<IPoseTimeseriesService, PoseTimeseriesService>();
 builder.Services.AddScoped<ILastMissionRunService, LastMissionRunService>();
-
+builder.Services.AddScoped<IInspectionService, InspectionService>();
```

Comment:
This should be InspectionFindingsService right?

Reply:
No we are using inspection service for both Inspection and InspectionFindings so far, as we only need to have an inspection finding when we have an inspection the current approach requires less code and not an additional file.

Reply:
Removed redundant line, hence resolved.